src/api-v1.js

```diff
@@ -580,6 +580,105 @@ api.declare({
 
 api.declare({
   method: 'put',
+  route: '/ami-set/:amiSetId',
+  name: 'createAmiSet',
+  deferAuth: true,
+  scopes: [['aws-provisioner:manage-ami-set:<amiSetId>']],
+  input: 'create-ami-set-request.json#',
+  output: 'get-ami-set-response.json#',
```

Comment:
These (input and output) point to the schemas we talked about today.  You can just omit these lines for the moment, and then return to add schemas in a later pull request.